docs/containerd-2.0.md

```diff
@@ -0,0 +1,105 @@
+# containerd 2.0
+
+Please try out the release binaries and report any issues at <https://github.com/containerd/containerd/issues/>.
+
+## What's new
+
+### Adds top level configuration migration support
```

Comment:
For this one I'd emphasize the new config version (3, not 2), and then the migration would be just part of the description.